app/templates/relay/_temperatures.html

```diff
@@ -1,8 +1,24 @@
-{% for t in temperatures %}
-<li>{{ t.hub_time }} <code>{{ t.hub_id }}</code> <code>{{ t.cell_id }}</code>
-  {% if t.relay %}<b>{{ t.temperature }}&deg;</b>{% else %}<s>{{ t.temperature }}&deg;</s>{% endif %}
-  <i>{{ t.sleep_period }}</i>
-  (received at {{ t.time }})
-  {% if t.relayed_time %}(relayed at {{ t.relayed_time }}){% endif %}
-</li>
-{% endfor %}
+<table>
+  <th>Hub Time</th>
```

Comment:
technically `<th>` has to be inside `<tr>`, even though browsers don't seem to care, but may as well stick to the spec